src/models/index.js

```diff
@@ -1,7 +1,56 @@
-import Product from './Product';
-import User from  './User';
+import Sequelize from 'sequelize';
+import createUser from './User'; 
+import createProduct from './Product'; 
 
-export {
-    Product,
-    User,
+const DATABASE = 'MarketDB';
+const DATABASE_USER = 'postgres';
+const DATABASE_PASSWORD = 'postgres';
+
+const sequelize = new Sequelize(
+  DATABASE,
+  DATABASE_USER,
+  DATABASE_PASSWORD,
+  {
+    dialect: 'postgres'
+  },
+);
+
+const models = {
+  User: createUser(sequelize, Sequelize),
+  Product: createProduct(sequelize, Sequelize),
+}
+
+Object.keys(models).forEach(key => 'associate' in models[key] && models[key].associate(models));
+
+async function fillDB (models) {
```

Comment:
To separate file

Reply:
Also why do you need it when sequelize already has `seeders-path`?

Reply:
Just forgot to delete